### ISSUE TITLE
Error: Member not found: 'MediaQuery.textScalerOf'.

### ISSUE THREAD

**Issue by tudosxxx** [syncfusion/flutter-widgets#1573]
syncfusion_flutter_datepicker-23.2.7/lib/src/date_picker/date_picker.dart:5630:35: Error: Member not found:
'MediaQuery.textScalerOf'.

MacBook-Pro ~ % flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.13.5, on macOS 13.5.2 22G91 darwin-x64,

**Comment by Prathap-Aosta** [comment]
Use flutter upgrade force
flutter upgrade --force

**Comment by LavanyaGowtham2021** [comment]
We have resolved the issue already and so we kindly request you to upgrade the latest version.

**Comment by gildonghong** [comment]
> lready and so we kindly request you to upgrade the latest ve

my flutter version 3.13
and I update this plugin to 24.2.7.
but same situaion as below.
error G75B77105: Member not found: 'MediaQuery.textScalerOf'.

How can I solve this?

**Comment by Yuvaraj-Gajaraj** [comment]
Hi @gildonghong ,

Due to the Flutter SDK version 3.16 breaking changes in textScaleFactor, we have migrated our source which is compatible with the latest version of Flutter SDK version 3.19. To Resolve this issue we suggest you upgrade the Flutter SDK to the latest version.

We have shared the related breaking changes link below for your reference.

Breaking changes, https://docs.flutter.dev/release/breaking-changes/deprecate-textscalefactor

Regards,
Yuvaraj.
